### ISSUE TITLE
[Keyboard navigation – LCS – Performance]: Keyboard focus is not visible on 'Graph and Download CSV  button' under Performance metrics pane.

### ISSUE THREAD

**Issue by mohit406** [highcharts/highcharts#15474]
**User Experience:** 
If the keyboard focus indicator is not visible on the controls than keyboard users will not be able to locate the focus on the page and will face difficulties while interacting with it.

**Note:**
User credentials should NOT be included in the bug.

**Repro Steps:**
Open #Url:https://lcs.dynamics.com
Login with valid credentials.
Select "Accessibility Review - D365FOE (Microsoft Corporation)" Project.
Project dashboard screen starts appearing.
Navigate to "Environment" section and select "Full details" button.
New screen starts appearing. Navigate to "Monitoring" section of Main page and select "Environment Monitoring".
New screen starts appearing where "Overview" tab is selected. Select "SQL Insights" tab.
New screen starts appearing where "Live View" tab is selected. Select "Performance Metrics" tab.
Navigate to metrics control and press tab.
Verify the issue.

**Actual Result:**
Keyboard focus is not visible on 'Graph and Download CSV button' under Performance metrics pane. When keyboard focus is on metrics  control and user press tab , focus is not visible on 'Graph and Download button'.

**Expected Result:**
Keyboard focus should visible on 'Graph and Download CSV button' under Performance metrics pane. When keyboard focus is on metrics  control and user press tab , focus should visible on 'Graph' and after pressing tab when focus is on graph chart, focus should visible on 'Download CSV button'.

**MAS Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html

**Test Environment:**
Version: 20H2 (OS Build: 19042.746)
Browser: Chrome (Version 88.0.4324.104)

#URL:[https://lcs.dynamics.com](url)

**Comment by mohit406** [comment]
![MAS2 4 7_Focus visible](https://user-images.githubusercontent.com/82201935/114141260-ff8e6700-992e-11eb-8761-5c747749bfa9.jpg)

**Comment by oysteinmoseng** [comment]
Hi @mohit406, we do not have access to this page for debugging, but the "Download CSV" link is not part of Highcharts. The chart should have clear keyboard focus by default if using the accessibility module as described in our documentation here: https://www.highcharts.com/docs/accessibility/accessibility-module. Also see https://api.highcharts.com/highcharts/accessibility.keyboardNavigation.focusBorder for options to customize the focus indicator of the charts.

Closing this for now, feel free to comment if we should reopen.
